core-java-collections/src/main/java/com/baeldung/list/multidimentional/ArrayListOfArrayList.java

```diff
@@ -0,0 +1,37 @@
+package com.baeldung.list.multidimentional;
+
+import java.util.ArrayList;
+
+public class ArrayListOfArrayList {
+	
```

Comment:
indent with spaces, not tabs -- see the formatting guidelines for more info, and consider importing the standard formatter into your IDE (see the eclipse directory of the tutorials repo if using eclipse)